### ISSUE TITLE
No windows.h on Visual Studio 2015

### ISSUE THREAD

**Issue by DJE98** [olilarkin/wdl-ol#62]
The opened project can't find the windows.h file. 

Message:

1>------ Erstellen gestartet: Projekt: IPlugEffect-app, Konfiguration: Debug Win32 ------
1>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Platforms\Win32\PlatformToolsets\v140_xp\Toolset.targets(36,5): warning MSB8003: Could not find WindowsSdkDir_71A variable from the registry.  TargetFrameworkVersion or PlatformToolset may be set to an invalid version number.
1>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.CppBuild.targets(366,5): warning MSB8003: Could not find WindowsSDKDir variable from the registry.  TargetFrameworkVersion or PlatformToolset may be set to an invalid version number.
1>  IPlugEffect.cpp
1>c:\users\eiser\documents\libraries\wdl-ol-master\wdl\mutex.h(37): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "windows.h": No such file or directory
1>  app_main.cpp
1>c:\users\eiser\documents\libraries\wdl-ol-master\iplugexamples\iplugeffect - kopie\app_wrapper\app_main.h(22): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "windows.h": No such file or directory
1>  app_dialog.cpp
1>c:\users\eiser\documents\libraries\wdl-ol-master\iplugexamples\iplugeffect - kopie\app_wrapper\app_main.h(22): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "windows.h": No such file or directory
1>  RtMidi.cpp
1>c:\users\eiser\documents\libraries\wdl-ol-master\wdl\rtaudiomidi\rtmidi.cpp(1952): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "Windows.h": No such file or directory
1>  RtAudio.cpp
1>c:\users\eiser\documents\libraries\wdl-ol-master\wdl\rtaudiomidi\rtaudio.h(509): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "windows.h": No such file or directory
1>  IPlugStandalone.cpp
1>c:\users\eiser\documents\libraries\wdl-ol-master\wdl\mutex.h(37): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "windows.h": No such file or directory
1>  asiolist.cpp
1>c:\users\eiser\documents\libraries\wdl-ol-master\asio_sdk\asiolist.cpp(1): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "windows.h": No such file or directory
1>  asiodrivers.cpp
1>c:\users\eiser\documents\libraries\wdl-ol-master\asio_sdk\asiodrivers.h(12): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "windows.h": No such file or directory
1>  asio.cpp
1>c:\users\eiser\documents\libraries\wdl-ol-master\asio_sdk\asio.cpp(32): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "windows.h": No such file or directory
1>  Code wird generiert...
========== Erstellen: 0 erfolgreich, 1 fehlerhaft, 2 aktuell, 0 übersprungen ==========

If you add  $(UniversalCRT_IncludePath) and $(UniversalCRT_LibraryPath_x86) to the visual studio configurations, you will get the same error (https://blogs.msdn.microsoft.com/vcblog/2015/03/03/introducing-the-universal-crt/).

Thank you for your effort in advance.

DJE98

**Comment by Soundbytes** [comment]
The app project in some/maybe all of the sample solutions seems to be broken.

I have been able to fix it through these steps:
- Load solution
- open property dialog of the -app project
- select 'All Configurations' from the configurtions dropdown
- go to the 'General' page
- For Target Version enter '8.1' (without the quotation marks)
- As Platform Toolset select Visual Studio 2015 (v140)
The last step might not be required. I did it anyways because I do not support WinXP anymore.

Now the Windows app Project should compile without an error.
